release-notes/20221014-air-gapped.md

```diff
@@ -0,0 +1,17 @@
+<!-- markdownlint-disable MD041 -->
+Submariner gateway pod tries to auto-discover the `public-ip`
+associated with the Gateway node by making API calls to some
+external servers in the internet. In an air-gapped deployment,
+these API calls are bound to fail and would lead to deployment
+failures.
+
+Subctl join command now supports `--air-gapped` option that
+instructs the Submariner gateway pod to use an empty `public-ip`
+in the local endpoint and avoid making any API calls to the external
+servers. The assumption here is that cluster is reachable on the
+private-ip. In case the cluster is not reachable on the private-ip
+but on some other `ip-address`, it can still be configured as a
+label on the Gateway node as follows, and the same would be used
+as the public-ip for the local endpoint.
+
+kubectl label node $Gateway `gateway.submariner.io/public-ip=ipv4:ip-address`
```

Comment:
This seems too much detail for a release note. I'd think most of this should be part of the permanent docs somewhere.

```suggestion
Subctl join command now supports a `--air-gapped` option that
instructs Submariner not to access any external servers.
```